### ISSUE TITLE
--where with --explore doesn't restrict the tests explored

### ISSUE THREAD

**Issue by ojintoad** [nunit/nunit-console#245]
From testing locally, the --where and --explore options do not seem to intersect the way you'd expect. TBF [the documentation doesn't explicitly say they work together](https://github.com/nunit/docs/wiki/Console-Command-Line), but it seems like the --explore option will always explore all tests in the passed in assembly regardless of the test selection language passed in with the where clause.

For example, the command:

```
.\packages\NUnit.Console.3.0.1\tools\nunit3-console.exe Tests\bin\Debug\Tests.dll --explore --where "test==Full.Name.Of.A.Test"
```

provides the same output as

```
.\packages\NUnit.Console.3.0.1\tools\nunit3-console.exe Tests\bin\Debug\Tests.dll --explore
```

Removing --explore in the former command only executes one test though.

This may be technically a feature request.

As indicated, I only tested with 3.0.1 but will verify if an updated console version has the same behavior.

**Comment by CharliePoole** [comment]
This is actually a framework bug and duplicates nunit/nunit#1143. That issue was fixed in NUnit 3.7. It shouldn't matter what version of the Console Runner you are using.

I'm closing this as a duplicate, but if you __are__ using 3.7+ we can re-examine it.

**Comment by ojintoad** [comment]
Thanks, we are in the process of upgrading so I'll follow up if necessary.

Also, note to self to look through closed GitHub tickets as well as open ones before opening issues.
